test/smoke/src/helpers/setupEnvironmentHelper.ts

```diff
@@ -14,8 +13,8 @@ const request = require("request");
 const source = require("vinyl-source-stream");
 const https = require("https");
 const fs = require("fs");
-const cp = require("child_process");
 const rimraf = require("rimraf");
+const cp = require("child_process");
```

Comment:
Looks like a redundant change

Reply:
It is a long story :)
But I can revert it if needed.

Reply:
@SounD120 Please, do it :)